test/api/v3/integration/challenges/POST-challenges.test.js

```diff
@@ -314,5 +314,33 @@ describe('POST /challenges', () => {
       groupLeader = await groupLeader.sync();
       expect(groupLeader.achievements.joinedChallenge).to.be.true;
     });
+
+    it('successfully sets a summary when summary provided', async () => {
+      let name = 'Test Challenge'
+      let summary = 'Test Summary'
```

Comment:
You're missing semi-colons after those two lines and a similar line below, and that's causing a failure in the tests (https://travis-ci.org/HabitRPG/habitica/jobs/295341692 ). There's a bunch of other failures in that link too although maybe some of them are cascading on from earlier ones. See which ones you can fix easily and then either push a new commit to this PR or run the lint tests locally ([Using Your Local Install to Modify Habitica's Website and API](http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica's_Website_and_API) has details). the lint tests generally run well locally even if the others don't.

I don't think this is related to the failing test you've asked about in the PR but clearing these up first will make the other errors, if any, easier to focus on.